src/ComputeSharp.D2D1/Shaders/Interop/D2D1ResourceTextureManager.cs

```diff
@@ -0,0 +1,487 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using ComputeSharp.D2D1.Shaders.Interop.Effects.ResourceManagers;
+using TerraFX.Interop.DirectX;
+using TerraFX.Interop.Windows;
+
+namespace ComputeSharp.D2D1.Interop;
+
+/// <summary>
+/// Provides an <c>ID2D1ResourceTextureManager</c> implementation, which can be used to manage resources in
+/// an effect created with <see cref="D2D1PixelShaderEffect"/> and share them across multiple effect instances.
+/// </summary>
+/// <remarks>
+/// <para>
+/// The built-in <see href="https://docs.microsoft.com/en-us/windows/win32/api/d2d1_1/nn-d2d1_1-id2d1effect"><c>ID2D1Effect</c></see>
```

Comment:
technically this is an implementation of `ID2D1EffectImpl`, not `ID2D1Effect` ;)

Reply:
Fixed in 28ed19c349592ecaafaafd00328df254e7556db9.